docs/cli/start.md

```diff
@@ -51,6 +51,10 @@ If you do not pass an argument to this command, you are prompted to select a nat
 `--port`
 * Port on which the local packager should listen on *(default: 8081)*
 
+`--resetCache`\
```

Comment:
Minor suggestion (no need to change here, just for next time): When using _two_ `LF`, the `\` is _not_ needed. It's only necessary when the source markdown has _one_ `LF` (to create a single line break in the rendered document).